### DISCUSSION TITLE
The image you selected does not appear to contain your school name

### DISCUSSION THREAD

**Discussion by ELJoOker2004** [community/community#77442] [GitHub Education]
### Hi everyone,

I uploaded my college id 9 times , and always rejected for 3 reasons , wrong first and last name , and no school name on the id . that's because the ai who idintify this is stupid and can't read arabic. and github support doesn't accept tickets about this subject

**Comment by hackelite01** [comment]
hey @ELJoOker2004 , I can do it easily. Drop the details on mail at [hackelite.sup@gmail.com](mailto:hackelite.sup@gmail.com)
